### ISSUE TITLE
Can you add Notification Support?

### ISSUE THREAD

**Issue by JosephSanjaya** [JessYanCoding/ProgressManager#66]
Thank you for such a great library, this is the best download progress listener that I could find, can you add notification support? so it can be more superb?

Thank you sir

**Comment by JessYanCoding** [comment]
What function do you want to achieve，ProgressListener is  Observer pattern

**Comment by JosephSanjaya** [comment]
Yeah, I mean add a sample or Base Class for this ProgressListener to be used in Download Notification, to show progress and custom Pending Intent.
